data/data/azure/variables-azure.tf

```diff
@@ -25,7 +25,7 @@ variable "azure_bootstrap_vm_type" {
 
 variable "azure_master_vm_type" {
   type = string
-  description = "Instance type for the master node(s). Example: `Standard_DS4_v3`."
+  description = "Instance type for the master node(s). Example: `Standard_D8s_v3`."
```

Comment:
In that case, could you also update the description for the bootstrap node on [line 23](https://github.com/openshift/installer/pull/4971/files#diff-383bd473843d54f24fdea79ed74893eccb34e213fda1595142bc3ac1d206313fR23)?